z390test/README.md

```diff
@@ -0,0 +1,243 @@
+# z390test
+
+This folder contains a framework for testing the z390 application.
+
+This is a black box test framework and does not perform unit testing
+of the actual z390 code. It calls the z390 app via a shell command.
+
+The idea is that you can run z390 with input and then check the output 
+from the process.
+
+
+## Setup
+
+Build the z390 Java jar and libraries for use by the tests. From the z390 root:
+
+    ./build.sh
+    win> BUILD.BAT
+
+## Run 
+
+By default the tests will not show the output of the `printOutput` method (see below).
+While developing tests, it is useful to see this output.
+
+    export Z390_PRINT_OUTPUT 1
+    win> SET Z390_PRINT_OUTPUT=1
+
+Run the tests from z390test clone directory
+
+    cd z390test
+    ./gradlew test
+    win> gradlew test
+
+If you use an IDE like IntelliJ or Eclipse, they have support for Gradle test suites.
+
+https://www.jetbrains.com/help/idea/work-with-tests-in-gradle.html
+
+
+    
+## Writing test cases
+
+The following is a basic test case that executes an assembly of a module in the z390 codebase 
+and checks the return code.
```

Comment:
Suggestion only. Since this is the reader's first exposure to the process. make this wordier. Sample: "Module TESTINS1.MLC resides in the z390 tests subdirectory. The following is a basic test case that executes an assembly of the module and checks the return code."

Reply:
Will updated as suggested.